src/layer.ts

```diff
@@ -113,13 +113,12 @@ export function applyExtensionLayer(service: Service, handlers: FunctionInfo[],
     if (handler.type === RuntimeType.UNSUPPORTED) {
       continue;
     }
-    const { runtime } = handler;
     const architecture =
       (handler.handler as any).architecture ?? (service.provider as any).architecture ?? DEFAULT_ARCHITECTURE;
     let extensionLayerARN: string | undefined;
     let extensionLayerKey: string = "extension";
 
-    if (architecture === ARM64_ARCHITECTURE && runtime && runtime in armRuntimeKeys) {
+    if (architecture === ARM64_ARCHITECTURE) {
```

Comment:
@duncanista , I have a question about this line. Will `architecture` always be `ARM64_ARCHITECTURE` if `runtime` is in `armRuntimeKeys`? If so, does it make sense to get rid of `armRuntimeKeys` for being redundant? And if not, does it make sense to add `nodejs14.x` to `armRuntimeKeys`?

I think this PR is probably good, I think I'm just a little confused about `armRuntimeKeys` and the nodejs14.x runtime.

Reply:
Not necessarily. Although every runtime in the `armRuntimeKeys` will always have an `ARM64_ARCHITECTURE`. Not every architecture will have a runtime in that map. `armRuntimeKeys` is needed since some runtimes, like `python3.8` and `python3.9`, need its specific ARM library layer. For example, `python3.7` and node runtimes do not need it since their library works right out of the box. Therefore, we don't need to add `nodejs14.x` or any other node environment to `armRuntimeKeys`.